### DISCUSSION TITLE
Shortcuts between data models and entries

### DISCUSSION THREAD

**Discussion by lukas-schaetzle** [directus/directus#4251] [Feature Requests]
I suggest adding 2 shortcut buttons: one for accessing the related data model from the entries. This should only be shown if the user has the rights to edit it.
![Screenshot_20210222_174845](https://user-images.githubusercontent.com/45203603/108891700-50f8c600-760f-11eb-801c-d5f2fda87a86.png)

And one from the data model to the related entries.
![Screenshot_20210222_175509](https://user-images.githubusercontent.com/45203603/108891725-58b86a80-760f-11eb-84fe-3bdf59feb726.png)

This is a rather simple feature request but I feel like it could make the UI navigation a bit more intuitive. I can create a pull request for it if nothing speaks against it.

**Comment by rijkvanzanten** [comment]
I'm wondering how useful this is after "initial setup" 🤔 

There's a window of time right after you've installed Directus where you're setting up the project, and going back and forth between these two pages. However, after that initial "configuration time", I'm wondering how useful this will be

Besides, seeing that settings defaults to data model, we'd only save one click. From the data-model page, the amount of clicks to get to the collection entries would be the same (1 on triple dot+"Show Entries" vs "Collection Module"+Collection) 🤔

**Reply by lukas-schaetzle** [reply]
You're right that the reduction of clicks is marginal. But I think the benefit here is that users do not have to find the relevant item in a new list on another page when they have already picked it before. (of course, you could use the search but that would take more clicks). Also, I think new users may expect these links as they seem quite logical.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
